### DISCUSSION TITLE
Add GaussianIntegrated/Gamma/InverseGamma LogLikelihoods

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1597] [Ideas/Science]
The benefit of these opposed to GaussianIntegratedUniformLogLikelihood is that the analytic derivative is stabler.

**Comment by MichaelClerx** [comment]
- [ ] GaussianIntegratedLogLikelihood
- [ ] GammaIntegratedLogLikelihood
- [ ] InverseGammaIntegratedLogLikelihood
